Doc/library/dataclasses.rst

```diff
@@ -46,7 +46,7 @@ directly specified in the :class:`!InventoryItem` definition shown above.
 Module contents
 ---------------
 
-.. decorator:: dataclass(*, init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False, match_args=True, kw_only=False, slots=False, weakref_slot=False)
+.. decorator:: dataclass(cls=None, /, *, init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False, match_args=True, kw_only=False, slots=False, weakref_slot=False)
```

Comment:
I think we should only expose the decorator interface and not the simple callable interface as well because `dataclass(**kwargs)(something) == dataclass(something, **kwargs)`.

Reply:
so i should revert this `dataclass` changes, right?

Reply:
Yes.